ochap-backend/src/main/resources/templates/shopping-cart-template.html

```diff
@@ -0,0 +1,365 @@
+                                    <ul class="Menu_footer">
+                                        <li class="active"><a href="#">My account</a></li>
+                                        <li><a href="#">Wishlist</a></li>
+                                        <li><a href="#">My Cart</a></li>
+                                        <li><a href="#">Checkout</a></li>
+                                        <li><a href="#">Login</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                            <div class="col-lg-3 col-md-6 col-sm-6">
+                                <div class="address">
+                                    <h3>corporation</h3>
+                                    <ul class="Links_footer">
+                                        <li class="active"><a href="#">My account</a></li>
+                                        <li><a href="#">Wishlist</a></li>
+                                        <li><a href="#">My Cart</a></li>
+                                        <li><a href="#"> Checkout</a></li>
+                                        <li><a href="#">Login</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+
+                            <div class="col-lg-3 col-md-6 col-sm-6 ">
+                                <div class="address">
+                                    <h3>why choose us</h3>
+                                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                                        incididunt ut labore et dolore magna </p>
+                                    <form class="newtetter">
+                                        <input class="tetter" placeholder="Your email" type="text" name="Your email">
+                                        <button class="submit">Subscribe</button>
+                                    </form>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="copyright">
+                <div class="container">
+                    <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a></p>
+                </div>
+            </div>
+        </div>
+
+    </footer>
+    <!-- end footer -->
+
+
+</div>
+
+<div class="overlay"></div>
+
+<!-- Javascript files-->
+<script src="js/jquery.min.js"></script>
+<script src="js/popper.min.js"></script>
+<script src="js/bootstrap.bundle.min.js"></script>
+<script src="js/jquery-3.0.0.min.js"></script>
+
+<!-- sidebar -->
+<script src="js/jquery.mCustomScrollbar.concat.min.js"></script>
+<script src="js/custom.js"></script>
+<script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
```

Comment:
Please serve these scripts from the static files. I think we already have a copy of this particular file. Else add it to the repository.